src/index.js

```diff
@@ -8,7 +8,7 @@ import { Circle } from './modules/circle.js'
 let canvas = document.getElementById("game");
 let ctx = canvas.getContext("2d");
 let inputHandler = new InputHandler();
-let player = new Player(0, "player", 40, 40, 20, 1, "red");
+let player = new Player(0, "player", 40, 40, 20, 2, "#f24bcb");
```

Comment:
Sadly we don't have a way yet to configure speeds per level. Maybe I can add that soon.

Until then, can you revert this line so there are no changes to this file?